website_and_docs/content/documentation/test_practices/encouraged/avoid_sharing_state.ja.md

```diff
@@ -19,3 +19,5 @@ aliases: [
 
 * テストごとに新しいWebDriverインスタンスを作成します。
 これにより、テストの分離が保証され、並列化がより簡単になります。
+
+    * もしテストランナーとして[pytest](https://pytest.org/)を選ぶ場合、グローバルフィクスチャでドライバーを返すことで簡単に実行できます。この方法を使えば、各テストが独自のドライバーインスタンスを取得し、テストが終了した後（成功・失敗にかかわらず）常にドライバーが終了することを保証できます.
```

Comment:
HI @shbenzer ,
This can be written in English and tagged for translation.
Even if it works perfectly with Google Translate, we can create issues requesting translations!

same for other translated pages too!

Reply:
I figured I'd try to ease the translation burden with LLM, but that's fair. I'll switch it to English.